packages/gestalt/src/PageHeader.js

```diff
@@ -0,0 +1,54 @@
+// @flow strict
+import React, { type Node, Children, type Element } from 'react';
+import Box from './Box.js';
+import Button from './Button.js';
+import IconButton from './IconButton.js';
+import Link from './Link.js';
+import Tooltip from './Tooltip.js';
+import Flex from './Flex.js';
+import Heading from './Heading.js';
+import Text from './Text.js';
+import { type Dimension } from './boxTypes.js';
+
+type Props = {|
+  title: string,
+  maxWidth?: Dimension,
```

Comment:
no action needed, but Dimension is just number or string, so we can pass anything else such as 'fit' etc.